.github/actions/localstack/action.yml

```diff
@@ -0,0 +1,39 @@
+name: 'LocalStack'
```

Comment:
this is another action to deploy a stack to localstack.

I got a bit crazy making this work - initially I added localstack as a service, and got what seemed to be communication issues (wrong ports?) but the log messages were completely misleading. This works now.